projects/1.19/assets/enders-delight/endersdelight/lang/zh_cn.json

```
@@ -0,0 +1,34 @@
+{
+  "block.endersdelight.chorus_crate": "箱装紫颂果",
+  "block.endersdelight.chorus_pie": "紫颂果派",
+  "block.endersdelight.endstone_stove": "末地石炉",
+  "block.endersdelight.stuffed_shulker_block": "填馅潜影贝",
+  "death.attack.wasWatered": "%1$s被浇了水",
+  "effect.endersdelight.phasing": "末影折跃",
+  "endersdelight.block.stuffed_shulker.use_shulker_bowl": "你需要一个碗才能吃这个",
+  "item.endersdelight.chorus_juice": "紫颂果汁",
+  "item.endersdelight.chorus_pie_slice": "紫颂果派切片",
+  "item.endersdelight.chorus_stew": "紫颂炖菜",
+  "item.endersdelight.chorus_stew_wood": "紫颂炖菜",
```

```suggestion
  "item.endersdelight.chorus_stew": "紫颂煲",
  "item.endersdelight.chorus_stew_wood": "紫颂煲",
```
```suggestion
  "item.endersdelight.chorus_stew": "紫颂炖",
  "item.endersdelight.chorus_stew_wood": "紫颂炖",
```
上方为原版译法，下方为农夫乐事译法，推荐原版